queue/src/test/java/org/killbill/bus/TestInMemoryEventBus.java

```diff
@@ -35,7 +35,7 @@ public class TestInMemoryEventBus {
     private TestEventBusBase testEventBusBase;
     private PersistentBus busService;
 
-    @BeforeClass(groups = "fast")
+    @BeforeClass(groups = "fast", alwaysRun = true)
```

Comment:
🤔 Saw your comment regarding the NPE. ack for now, but I suspect something else is happening...

Reply:
Hmm.. maybe because it was mixed with "slow" and "fast" group? And now since every test in `TestInMemoryEventBus` is "fast", it should be fine without `alwaysRun = true` ? 

Let me run again several times locally and report here what happened.